lib/CreepPlayers/NecromancerHealPlayer.js

```diff
@@ -0,0 +1,23 @@
+
+var BaseCreepPlayer = require('./BaseCreepPlayer');
+var Spawn = require('./Behaviors/Spawn');
+var Run = require('./Behaviors/RunDumb');
+var KiteWithFriend = require('./Behaviors/KiteWithFriendDumb');
+var Heal = require('./Behaviors/Heal');
+
+
+function NecromancerPlayer(creepController) {
+    this.cc = creepController;
+
+    var spawn = new Spawn(this.cc);
+    var run = new Run(this.cc);
+    var heal = new Heal(this.cc);
+    var kite = new KiteWithFriend(this.cc);
+
+    this.behaviors = [heal, spawn, kite, run];
```

Comment:
potentially depending on the stats of the monster, the heal behavior only heals if the unit has less hp than their max - the amount they can heal. if they do sit there and heal forever, hopefully there are other monsters that are bothering the player.